src/collectors/prometheus_rabbitmq_core_metrics_collector.erl

```diff
@@ -334,6 +339,8 @@ label({P, {#resource{virtual_host = QVHost, kind = queue, name = QName},
     %% channel_queue_exchange_metrics {channel_id, {queue_id, exchange_id}}
     [{channel, P}, {queue_vhost, QVHost}, {queue, QName},
      {exchange_vhost, EVHost}, {exchange, EName}];
+label({B1, B2}) when is_binary(B1), is_binary(B2) ->
```

Comment:
When per user auth attempts are disabled, and per object metrics are enabled, there should be no `ip` or `username` label.

Red is what we have currently, green is what I think we should have. What do you think @dcorbacho?

```diff
# TYPE rabbitmq_auth_attempts_total counter
# HELP rabbitmq_auth_attempts_total Total number of authorization attempts on a node
- rabbitmq_auth_attempts_total{ip="",username=""} 42
+ rabbitmq_auth_attempts_total 42
# TYPE rabbitmq_auth_attempts_succeeded_total counter
# HELP rabbitmq_auth_attempts_succeeded_total Total number of successful authorization attempts on a node
- rabbitmq_auth_attempts_succeeded_total{ip="",username=""} 42
+ rabbitmq_auth_attempts_succeeded_total 42
# TYPE rabbitmq_auth_attempts_failed_total counter
# HELP rabbitmq_auth_attempts_failed_total Total number of failed authorization attempts on a node
- rabbitmq_auth_attempts_failed_total{ip="",username=""} 0
+ rabbitmq_auth_attempts_failed_total 0
```